lib/rules/jsx-first-prop-new-line.js

```diff
@@ -15,14 +17,49 @@ module.exports = {
       category: 'Stylistic Issues',
       recommended: false
     },
+    fixable: 'code',
 
-    schema: [{
-      enum: ['always', 'never', 'multiline', 'multiline-multiprop']
-    }]
+    schema: [
+      {
+        enum: ['always', 'never', 'multiline', 'multiline-multiprop']
+      },
+      {
+        oneOf: [{
```

Comment:
indent should not be passed separately - it should just use 4 spaces (the `indent` default) and the individual indent rules should handle autofixing them.